starter_code/src/movies.js

```diff
@@ -1,20 +1,71 @@
 /* eslint no-restricted-globals: 'off' */
-// Turn duration of the movies from hours to minutes 
+// Turn duration of the movies from hours to minutes
 
+// Get the average of all rates with 2 decimals
 
-// Get the average of all rates with 2 decimals 
+function ratesAverage(array) {
+  var averageRate = array.reduce(function(sum, oneMovie) {
+    var totalRate = sum + parseInt(oneMovie.rate);
```

Comment:
you can  change parseInt here to Number in order to pass one of the tests for the next iteration 

``` var totalRate = sum + Number(oneMovie.rate);```

There is a small [difference](https://stackoverflow.com/questions/4090518/what-is-the-difference-between-parseint-and-number) between them